resources/authgear/templates/en/web/authflowv2/settings_picture_edit.html

```diff
@@ -1,7 +1,52 @@
 {{ template "authflowv2/__settings_page_frame.html" . }}
 
 {{ define "page-content" }}
-<div>
-  testtt
+
+
+{{ template "authflowv2/__navbar.html"
+(dict
+  "BackHref" (call $.MakeURL "/settings/profile")
+  "BackTitle" (include "v2.component.navbar.default.item-back-button-label" nil )
+  "Title" (translate "v2.page.settings-profile.default.profile-picture-title" nil)
+)
+}}
```

Comment:
I updated the structure of rendering navbar. Please help update it.

https://github.com/cychiuae/authgear-server/blob/feat-authuiv2-settings/resources/authgear/templates/en/web/authflowv2/settings_profile.html